testing/test_gateway.py

```diff
@@ -363,6 +365,7 @@ def test_status_with_threads(self, makegateway):
         # is already closed.
         for i in range(10):
             rstatus = gw.remote_status()
+            time.sleep(0.1)
             if rstatus.numexecuting == 0:
                 return
```

Comment:
i'd do the time.sleep() as the last command in the for-body, i.e. after the "return". 
i think we could merge this one then.